<launch>  
  <node pkg="steerforce_conv" type="steerforce_converter.py" name="steerforce_converter" output="screen">
    <rosparam file="$(find steerforce_conv)/params/params.yaml" />
  </node>
</launch>